app/src/renderer/js/main.js

```diff
@@ -40,23 +40,27 @@ document.addEventListener('DOMContentLoaded', () => {
   const windowHeader = document.querySelector('.window-header');
 
   const [micOnIcon, micOffIcon] = toggleAudioRecordBtn.children;
-
+  
+  
   // Initial variables
-  let lastValidInputWidth = 512;
-  let lastValidInputHeight = 512;
-  const dimensions = {
-    height: 512,
-    width: 512,
-    ratio: '1:1',
-    ratioLocked: false
-  };
+  const dimensions = app.kap.settings.get('dimensions');
+  const { width, height, ratioLocked } = dimensions;
+  let lastValidInputWidth = width;
+  let lastValidInputHeight = height;
 
   // Init dynamic elements
   if (app.kap.settings.get('recordAudio') === true) {
     toggleAudioRecordBtn.classList.add('is-active');
     micOnIcon.classList.remove('hidden');
     micOffIcon.classList.add('hidden');
   }
+    
+  // Set initial values
+  [inputWidth.value, inputHeight.value] = [width, height].map(Math.round);
```

Comment:
Haha clever 😍

Reply:
I think it's too clever. This is much more readable and only 4 characters more:

```js
inputWidth.value = Math.round(width);
inputHeight.value = Math.round(height);
```

Reply:
Do you want me to change it?

Reply:
Yes, please :)

Reply:
0fbd884